README.rst

```diff
@@ -178,24 +178,10 @@ Look at the new Repository Version created
     }
 
 
-Publish a Repository Version and create a Publication
------------------------------------------------------
-
-``$ http POST :24817/pulp/api/v3/docker/publish/ repository=$REPO_HREF``
-
-.. code:: json
-
-    {
-        "task": "/pulp/api/v3/tasks/fd4cbecd-6c6a-4197-9cbe-4e45b0516309/"
-    }
-
-``$ export PUBLICATION_HREF=$(http :24817/pulp/api/v3/publications/ | jq -r '.results[0] | ._href')``
-
-Add a Docker Distribution to serve your publication
----------------------------------------------------
-
-``$ http POST http://localhost:24817/pulp/api/v3/docker-distributions/ name='baz' base_path='foo' publication=$PUBLICATION_HREF``
+Add a Docker Distribution to serve the latest Repository Version
```

Comment:
s/Add a Docker Distribution to serve the latest Repository Version/Add a Docker Distribution to serve a Repository Version

Can we also add a note how that works? if repository is provided that latest version is automatically taken unless a repository_version is specified?

Reply:
I'll add example for using both a repository and a repository version. I'll explain the difference here also.